foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/sessions/Project.java

```diff
@@ -49,10 +49,12 @@
 import org.eclipse.persistence.internal.helper.ConcurrentFixedCache;
 import org.eclipse.persistence.internal.identitymaps.AbstractIdentityMap;
 import org.eclipse.persistence.internal.identitymaps.IdentityMap;
+import org.eclipse.persistence.internal.jpa.jpql.HermesParser;
```

Comment:
this goes against keeping the dependency on this optional (ie in jpms descriptor)